### PR TITLE
For #2273

### PR DESCRIPTION
**Pull request by mugitty**
Implemented gorule-0000064

### CODE REVIEW

**Review by kltm — COMMENTED** [review]
Quick review looks good to me, but I'd like to defer the final call/merge/release to @dustine32

File: bin/ontobio-parse-assocs.py [hunk 1]
```diff
@@ -143,9 +143,11 @@ def main():
         rule_set = assocparser.RuleSet.ALL
     
     goref_metadata = None
+    goref_species = None
     if args.metadata_dir:
         absolute_metadata = os.path.abspath(args.metadata_dir)
-        goref_metadata = metadata.yamldown_lookup(os.path.join(absolute_metadata, "gorefs"))    
+        goref_metadata = metadata.yamldown_lookup(os.path.join(absolute_metadata, "gorefs"))
+        goref_species = metadata.yaml_set(absolute_metadata, "go-reference-species.yaml", "taxon_id")
```
Comment (dustine32) [inline bin/ontobio-parse-assocs.py:150]: My nitpicky thing here that doesn't need to be fixed (and I'm very open to arguing whether it's actually wrong) is the naming of the "goref_species" variable. I think its name is derived from `goref_metadata` above, which points to GO_REF ID data. However, the species data is totally different from GO_REFs and I might be misled into thinking the `goref_species` var has something to do with GO_REF IDs.

Could we call it something like `reference_species_metadata` or `ref_species_metadata`? @mugitty What do you think? Ignore this if you like!

**Review by dustine32 — APPROVED** [review]
Looks good @mugitty. Will filter _out_ species in our reference species list.

File: bin/ontobio-parse-assocs.py (hunk 1 above)
Comment (mugitty) [reply bin/ontobio-parse-assocs.py]: @dustine32, I have updated the variable name

Comment (dustine32) [reply bin/ontobio-parse-assocs.py]: Thanks so much @mugitty!

**Pull request merged**